2021-finalproject/Koonyu/README.md

```diff
@@ -0,0 +1,7 @@
+**2021/10/5
+# 学习总结
+
+
+**目前为止，我们除了学习C++的部分用法，还初步学习shell命令行、bash脚本、vim编辑器、ssh在数据整理的应用、密码学和git的基本用法。
```

Comment:
Just in case, if you mean **bold** on this line, add two `*` at the end

Reply:
Got it